pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller.go

```diff
@@ -106,14 +110,17 @@ func (c *ssarController) sync(ctx context.Context, controllerContext factory.Syn
 	}
 	klusterlet = klusterlet.DeepCopy()
 
-	// if the ssar checking is already processing, ignore reconciling this turn.
+	// if the ssar checking is already processing, requeue it.
```

Comment:
why we need to requeue it? I think this is to avoid multiple checks happens

Reply:
this is for config secret update case, e.g. in our integration test,  we create the hub config secret then update it, for sar controller, when it handling the create event, the update event is coming, in this time, the update event will not be handled, so the update event will be missed

Reply:
increase the requeue time